R/language.R

```diff
@@ -33,10 +49,16 @@
 #' is_atomic_lang(quote(call()))
 #'
 #' # With the exception of names, atomic language objects have the
-#' # property of being self-quoting:
+#' # property of being self-quoting literals:
```

Comment:
So the first intuition was the right one :)

Now updated, along with some explanation of the role of pairlists in language objects.